docs/api/logstash-configuration-management/list-pipeline.asciidoc

```diff
@@ -6,6 +6,11 @@
 
 experimental[] List all centrally-managed Logstash pipelines.
 
+[IMPORTANT]
+====
+Limit your number of pipelines to 10k or fewer. When the number of pipelines surpasses 10k, you may see performance issues on Kibana.
+====
```

Comment:
```suggestion
IMPORTANT: Limit the number of pipelines to 10k or fewer. As the number of pipelines nears and surpasses 10k, you may see performance issues on Kibana.
```

Reply:
You can use the basic (not block) format for a single paragraph admonition. 

What do you think about saying "As the number of pipelines nears and surpasses..." to indicate that 10k is not an absolute?

Reply:
Oh thanks! TIL: didn't know without block it works. And thanks for revising the message.